src/GithubAPINew.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable */
+/* eslint-disable  */
+import React from 'react'
+import GitHubLogin from 'react-github-login'
+
+function GithubAPINew() {
+  const onSuccess = function(details) {
+    const code = details.code
+    console.log(code)
+
+    return fetch(
+      `https://gitfund-oauth.herokuapp.com/authenticate/${code}`
+    ).then((response) => console.log(response))
+    // fetch(
```

Comment:
Remove the commented statements if not required in the future

Reply:
Removed